src/test/scala/com/datastax/sparkstress/WriteTaskTests/WriteTaskTests.scala

```diff
@@ -15,112 +19,218 @@ class WriteTaskTests extends FlatSpec
-  useSparkConf(defaultSparkConf)
+  useSparkConf(sparkConf)
 
+  // Allow us to rerun tests with a clean slate
   val conn = CassandraConnector(Set(EmbeddedCassandra.getHost(0)))
   conn.withSessionDo { session =>
-    session.execute(s"""CREATE KEYSPACE IF NOT EXISTS test1 WITH REPLICATION = { 'class': 'SimpleStrategy', 'replication_factor': 1 } """)
-    session.execute(s"""CREATE KEYSPACE IF NOT EXISTS test2 WITH REPLICATION = { 'class': 'SimpleStrategy', 'replication_factor': 1 } """)
-    session.execute(s"""CREATE KEYSPACE IF NOT EXISTS test3 WITH REPLICATION = { 'class': 'SimpleStrategy', 'replication_factor': 1 } """)
-    session.execute(s"""CREATE KEYSPACE IF NOT EXISTS test4 WITH REPLICATION = { 'class': 'SimpleStrategy', 'replication_factor': 1 } """)
-    session.execute(s"""CREATE KEYSPACE IF NOT EXISTS test5 WITH REPLICATION = { 'class': 'SimpleStrategy', 'replication_factor': 1 } """)
-    session.execute(s"""CREATE KEYSPACE IF NOT EXISTS test6 WITH REPLICATION = { 'class': 'SimpleStrategy', 'replication_factor': 1 } """)
-    session.execute(s"""CREATE KEYSPACE IF NOT EXISTS test7 WITH REPLICATION = { 'class': 'SimpleStrategy', 'replication_factor': 1 } """)
+    session.execute(s"""DROP KEYSPACE IF EXISTS test1 """)
+    session.execute(s"""DROP KEYSPACE IF EXISTS test2 """)
+    session.execute(s"""DROP KEYSPACE IF EXISTS test3 """)
+    session.execute(s"""DROP KEYSPACE IF EXISTS test4 """)
+    session.execute(s"""DROP KEYSPACE IF EXISTS test5 """)
+    session.execute(s"""DROP KEYSPACE IF EXISTS test6 """)
+    session.execute(s"""DROP KEYSPACE IF EXISTS test7 """)
   }
+  Thread.sleep(5000)
+
+  conn.withSessionDo { session =>
+    session.execute(s"""CREATE KEYSPACE test1 WITH REPLICATION = { 'class': 'SimpleStrategy', 'replication_factor': 1 } """)
+    session.execute(s"""CREATE KEYSPACE test2 WITH REPLICATION = { 'class': 'SimpleStrategy', 'replication_factor': 1 } """)
+    session.execute(s"""CREATE KEYSPACE test3 WITH REPLICATION = { 'class': 'SimpleStrategy', 'replication_factor': 1 } """)
+    session.execute(s"""CREATE KEYSPACE test4 WITH REPLICATION = { 'class': 'SimpleStrategy', 'replication_factor': 1 } """)
+    session.execute(s"""CREATE KEYSPACE test5 WITH REPLICATION = { 'class': 'SimpleStrategy', 'replication_factor': 1 } """)
+    session.execute(s"""CREATE KEYSPACE test6 WITH REPLICATION = { 'class': 'SimpleStrategy', 'replication_factor': 1 } """)
+    session.execute(s"""CREATE KEYSPACE test7 WITH REPLICATION = { 'class': 'SimpleStrategy', 'replication_factor': 1 } """)
+  }
+
+  // Wipe content from DSEFS too
+  val dseFsClient = new DseFsClient(new DseFsClientConf(Seq(HostAndPort.defaultPublicDseFsEndpoint("localhost"))))
+  val parquetTableName = "parquet_test"
+  val textTableName = "text_test"
+  val jsonTableName = "json_test"
+  val csvTableName = "csv_test"
+  val datasetTestKS = "test2"
+  dseFsClient.deleteRecursive(FilePath(s"/$datasetTestKS.$parquetTableName"))
```

Comment:
I believe you have missed the remark about this one. `deleteRecursive` needs awaiting.

Reply:
I did miss this one, fixed.